website/content/docs/connect/gateways/api-gateway/index.mdx

```diff
@@ -1,28 +1,61 @@
 ---
 layout: docs
-page_title: API Gateways Overview
-description: API gateways are objects in Consul that enable ingress requests to services in your service mesh. Learn about API gateways for VMs in this overview.
+page_title: API gateways overview
+description: API gateways provide an ingress point for service mesh traffic. Learn how API gateways add listeners for external traffic and route HTTP requests to services in the mesh.
 ---
 
-# API gateway overview
+# API gateways overview
 
-API gateways enable external network clients to access applications and services
-running in a Consul datacenter. This type of network traffic is commonly
-called _north-south_ network traffic because it refers to the flow of
-data into and out of a specific environment. API gateways can also forward
-requests from clients to specific destinations based on path or request
-protocol.
+This topic provides overview information about API gateways in Consul. API gateways enable external network clients to access applications and services running in a Consul datacenter. Consul API gateways can also forward requests from clients to specific destinations based on path or request protocol.
+
+## API gateway use cases
 
 API gateways solve the following primary use cases:
 
-- **Control access at the point of entry**: Set the protocols of external connection
-  requests and secure inbound connections with TLS certificates from trusted
-  providers, such as Verisign and Let's Encrypt.
-- **Simplify traffic management**: Load balance requests across services and route
-  traffic to the appropriate service by matching one or more criteria, such as
-  hostname, path, header presence or value, and HTTP method.
-
-Consul supports API
-gateways for virtual machines and Kubernetes networks. Refer to the following documentation for next steps:
-- [API Gateways on VMs](/consul/docs/connect/gateways/api-gateway/usage)
-- [API Gateways for Kubernetes](/consul/docs/api-gateway).
+- **Control access at the point of entry**: Set the protocols of external connection requests and secure inbound connections with TLS certificates from trusted providers, such as Verisign and Let's Encrypt.
+- **Simplify traffic management**: Load balance requests across services and route traffic to the appropriate service by matching one or more criteria, such as hostname, path, header presence or value, and HTTP method.
+
+## Workflows
+
+You can deploy API gateways to networks that implement a variety of computing environments: 
+
+- Services hosted on VMs
+- Kubernetes-orchestrated service containers
+- Kubernetes-orchestrated containers in OpenShift
```

Comment:
@missylbytes is this the correct phrasing? I think it's not a blocker if you aren't able to respond by EOD, but I'm wondering if this is the right way to talk about OpenShift.

Reply:
I think this works and is a worthwhile distinction since OpenShift has a stricter set of requirements than vanilla k8s. If anything, I would change the OpenShift line to `service containers` to be consistent with the line above it